src/app/router/connectorRoutes.tsx

```diff
@@ -39,37 +39,37 @@ function getConnectorRoutes() {
   return [
     {
       path: "lnd",
-      element: <ConnectLnd />,
+      element: <ConnectLnd logo={lnd} />,
```

Comment:
What is the benefit of passing the logo to the component? each component could set the logo themselves since we have components for each connector type

Reply:
Good point about the logos already being there, but they are just imports. In my opinion I think this would be best to be changed, as it introduces unnecessary prop drilling which will make the components more difficult to maintain.

Reply:
Thanks @rolznz - now the components import the logo. 

By the way, the components ConnectLndHub and ConnectGaloy are examples of connectors sharing the same components using different logos.

Reply:
Thanks @taboca !